jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1302,10 +1306,13 @@ Function Start-Migration {
     Begin {
         Write-ToLog -Message:('####################################' + (get-date -format "dd-MMM-yyyy HH:mm") + '####################################')
         # Start script
-        $admuVersion = '2.3.0'
+        Write-ToLog -Message:('JumpCloud username param: ' + $JumpCloudUserName + ' Selected username param: ' + $SelectedUserName)
```

Comment:
```suggestion
```

This message is duplicated by line 1458 `        Write-ToLog -Message:('Windows Profile "' + $SelectedUserName + '" is going to be converted to "' + $localComputerName + '\' + $JumpCloudUsername + '"')
`